molecule/default/Dockerfile.j2

```diff
@@ -0,0 +1,8 @@
+{% if item.registry is defined %}
+FROM {{ item.registry.url }}/{{ item.image }}
+{% else %}
+FROM {{ item.image }}
+{% endif %}
```

Comment:
Isn't the registry URL part of the image name is you use something like `docker.io/pycontribs/centos:8`?

Reply:
This is whole file is actually a copy/paste-artifact and never used. So I removed it in https://github.com/elan-ev/opencast_mariadb/pull/1/commits/9a27f517a23058c9d76b0eb2f441f393e63e0765